### DISCUSSION TITLE
Updated organization name

### DISCUSSION THREAD

**Discussion by PartyDonut** [DonutWare/Fladder#31] [Announcements]
Changed the organization name to DonutWare. 
No real valid reason, was just anoying to always see the url with the dash in between Fladder-app. And its not just an app.

Make sure you update your remotes if applicable. The old link should keep working for now.

**Comment by bazzadazza72** [comment]
Should the AltStore repository name be changed as well? I submitted a PR (#37) adding the latest update and didn't think to update the name, my apologies 😅

**Comment by PartyDonut** [comment]
Yes completely forgot to change it there myself, I've updated the developerName in alstore.json. Thanks for reminding me 👍🏼
